createcoveragereport.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script merges all coverage files found under the 'coverage' directory
+# and then creates a dotcover XML report and from that creates a browsable
+# ReportGenerator html report.
+
+# Use an appropriate version of nuget... preferring
+# first an existing NUGET variable, then NuGet, then
+# just falling back to the path.
+if [ -z "$NUGET" ]
+then
+  if [ -n "$NuGet" ]
+  then
+    NUGET="$NuGet"
+  else
+    NUGET="nuget"
+  fi
+fi
+
+$NUGET install -Verbosity quiet -OutputDirectory packages -Version 2017.1.20170613.162720 JetBrains.dotCover.CommandLineTools
+$NUGET install -Verbosity quiet -OutputDirectory packages -Version 2.4.5.0 ReportGenerator
+
+DOTCOVER=$PWD/packages/JetBrains.dotCover.CommandLineTools.2017.1.20170613.162720/tools/dotCover.exe
+REPORTGENERATOR=$PWD/packages/ReportGenerator.2.4.5.0/tools/ReportGenerator.exe
+FIND=/usr/bin/find
+
+if [ ! -d coverage ]
+then
+ mkdir coverage
+fi
+
+echo "Merging reports..."
+
+echo "<MergeParams>" > "coverage/merge.xml"
+for filename in coverage/*.dvcr; do
+  echo "<Source>$(basename $filename)</Source>" >> "coverage/merge.xml"
+done
+echo "<Output>all.dvcr</Output>" >> "coverage/merge.xml"
+echo "</MergeParams>" >> "coverage/merge.xml"
+
+$DOTCOVER merge coverage/merge.xml
```

Comment:
Given the report part below, we could probably work out how to build the command line for merge, too... 

I think this should work:

```text
dotcover merge -Source=$"(echo coverage/*.dvcr | sed 's/ /;/g') -Output=coverage/all.dcvr ""
```